File: SamplePrograms/WaitFor3Seconds.c
#include <stdio.h>
#include <unistd.h>  // For the sleep function

int main() {
    // Wait for 3 seconds
    printf("Waiting for 3 seconds...\n");
    sleep(3);  // Sleep for 3 seconds
    
    // Print the message
    printf("All good to go!\n");
    
    return 0;
}

File: SamplePrograms/WaitFor3Seconds.java
public class WaitFor3Seconds {

    public static void main(String[] args) {
        try {
            // Wait for 3 seconds
            System.out.println("Waiting for 3 seconds...");
            Thread.sleep(3000);  // Sleep for 3000 milliseconds (3 seconds)
            
            // Print the message
            System.out.println("All good to go!");
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }
}
